### PR TITLE
Summary stats notebook for ships

### PR DESCRIPTION
**Pull request by HenryW95**
Closes AN-4475

Summary stats demo notebook for object detection (ships).
Removes spaces from notebook filenames in the jupyter-notebooks/analytics/user-guide directory

### CODE REVIEW

**Comment by BradNeuberg** [comment]
Heads up that the main README.md file for the notebook repo has changed from a recent PR that has landed (https://github.com/planetlabs/notebooks/pull/144). Make sure to rebase master into this PR to get these updates, and to add a link to the new summary stats notebook to the Analytics section of the README.md in the right section.

File: jupyter-notebooks/analytics/user-guide/05_summary_statistics_ships_and_clouds.ipynb [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create a column that estimates the percentage of imagery where the model is expected to perform.\n",
+    "df['Usable Percentage'] = (df['Clear Area (udm2_band_1)'] + df['Light Haze Area (udm2_band_4)'] + (df['Heavy Haze Area (udm2_band_5)'] / 2)) / df['Total Area (udm2)']\n",
```
Comment (strixcuriosus) [inline jupyter-notebooks/analytics/user-guide/05_summary_statistics_ships_and_clouds.ipynb:364]: You could make the `/2` a configurable variable, e.g. 
```
LIGHT_HAZE_FRAC = 1.0
HEAVY_HAZE_FRAC = 0.5
df['Usable Percentage'] = (df['Clear Area (udm2_band_1)'] + df['Light Haze Area (udm2_band_4)'] * LIGHT_HAZE_FRAC + (df['Heavy Haze Area (udm2_band_5)'] * HEAVY_HAZE_FRAC)) / df['Total Area (udm2)']
```

**Comment by strixcuriosus** [comment]
LGTM

File: jupyter-notebooks/analytics/user-guide/05_summary_statistics_ships_and_clouds.ipynb (hunk 1 above)
Comment (HenryW95) [reply jupyter-notebooks/analytics/user-guide/05_summary_statistics_ships_and_clouds.ipynb]: Great idea!

**Pull request merged**